package.json

```diff
@@ -106,7 +106,7 @@
     "libxmljs": "^0.18.0",
     "lodash": "^4.5.1",
     "lodash-deep": "^2.0.0",
-    "mapbox": "^1.0.0-beta5",
```

Comment:
We're already using beta7, this just cleans up the yarn.lock file a little (it had multiple versions of whatwg-fetch before I changed this).